### PR TITLE
[SPARK-6700] disable flaky test

### PR DESCRIPTION
**Pull request by davies**

### CODE REVIEW

**Comment by davies** [comment]
cc @andrewor14

File: yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala [hunk 1]
```diff
@@ -143,7 +143,8 @@ class YarnClusterSuite extends FunSuite with BeforeAndAfterAll with Matchers wit
     }
   }
 
-  test("run Python application in yarn-cluster mode") {
+  // Enable this once fix SPARK-6700
```
Comment (sryza) [inline yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala]: Is SPARK-6700 for disabling the flaky test or for the fix? It shouldn't be for both.

Comment (andrewor14) [inline yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala]: I think the issue just describes the fact that `YarnClusterSuite` is flaky. This is probably fine

**Comment by andrewor14** [comment]
Merging this in master 1.3 since it's breaking builds.

File: yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala (hunk 1 above)
Comment (sryza) [inline yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala]: Right, but does it make sense to have statement that saying "change this once SPARK-6700 is resolved" as part of SPARK-6700?

**Pull request closed without merging**